tests/console_test.go

```diff
@@ -75,6 +78,13 @@ var _ = Describe("[rfe_id:127][posneg:negative][crit:medium][vendor:cnv-qe@redha
 		return expecter, errChan
 	}
 
+	deleteDataVolume := func(dv *cdiv1.DataVolume) {
+		if dv != nil {
+			By("g the DataVolume")
```

Comment:
what does "g" stand for?

Reply:
Oops, should be "Deleting" - I'll fix that

Reply:
Done